src/main/java/io/anserini/search/SearchTweets.java

```diff
@@ -130,24 +132,24 @@ public static void main(String[] args) throws Exception {
     for ( MicroblogTopic topic : topics ) {
       long curQueryTime = System.nanoTime();
 
-      Query filter = LongPoint.newRangeQuery(StatusField.ID.name, 0L, topic.getQueryTweetTime());
-      Query query = AnalyzerUtils.buildBagOfWordsQuery(StatusField.TEXT.name, IndexTweets.ANALYZER, topic.getQuery());
+      Query filter = TermRangeQuery.newStringRange(FIELD_ID, "0", String.valueOf(topic.getQueryTweetTime()), true, true);
```

Comment:
I'd add a comment about why we're doing a range query, it's non-obvious (to me at least)